tests/maintain/test_index.py

```diff
@@ -105,3 +191,89 @@ def test_edge_index(self):
 
         resp = self.client.execute('CREATE EDGE INDEX disorder_edge_1_index ON edge_1(col3, col2)')
         self.check_resp_succeeded(resp)
+
+        # Insert some data
+        resp = self.client.execute('INSERT EDGE edge_1(col1, col2, col3, col4) VALUES \'101\'->\'102\':(\'Red\', 81, 45.3, \'2010-09-01 08:00:00\')')
+        self.check_resp_succeeded(resp)
+
+        resp = self.client.execute('INSERT EDGE edge_1(col1, col2, col3, col4) VALUES \'102\'->\'103\':(\'Yellow\', 22, 423.8, \'2011-09-01 08:00:00\')')
+        self.check_resp_succeeded(resp)
+
+        resp = self.client.execute('INSERT EDGE edge_1(col1, col2, col3, col4) VALUES \'103\'->\'101\':(\'Blue\', 91, 43.1, \'2010-09-01 12:00:00\')')
+        self.check_resp_succeeded(resp)
+
+        # # Rebuild Edge Index
+        # resp = self.client.execute_query('REBUILD EDGE INDEX single_edge_index')
+        # self.check_resp_succeeded(resp)
+
+        # resp = self.client.execute_query('REBUILD EDGE INDEX multi_edge_1_index')
+        # self.check_resp_succeeded(resp)
+
+        # resp = self.client.execute_query('REBUILD EDGE INDEX disorder_edge_1_index')
+        # self.check_resp_succeeded(resp)
+
+        # resp = self.client.execute_query('REBUILD EDGE INDEX non_existent_edge_index')
+        # self.check_resp.failed(resp)
+
+        # Show Edge Index Status
+        # resp = self.client.execute_query('SHOW EDGE INDEX STATUS')
+        # self.check_resp_succeeded(resp)
+        # self.check_out_of_order_result(resp, [['single_edge_index', 'SUCCEEDED'], ['multi_edge_1_index', 'SUCCEEDED'], ['disorder_edge_1_index', 'SUCCEEDED']])
+
+        # Describe Edge Index
+        resp = self.client.execute_query('DESC EDGE INDEX single_edge_index')
+        self.check_resp_succeeded(resp)
+        expect = [['col2', 'int64']]
+        self.check_result(resp, expect)
+
+        resp = self.client.execute_query('DESC EDGE INDEX multi_edge_1_index')
+        self.check_resp_succeeded(resp)
+        expect = [['col2', 'int64'],
+                  ['col3', 'double']]
+        self.check_result(resp, expect)
+
+        resp = self.client.execute_query('DESC EDGE INDEX disorder_edge_1_index')
+        self.check_resp_succeeded(resp)
+        expect = [['col3', 'double'],
+                  ['col2', 'int64']]
+        self.check_result(resp, expect)
+
+        resp = self.client.execute_query('DESC EDGE INDEX non_existent_edge_index')
+        self.check_resp_failed(resp)
+
+        # Show Create Edge Index
+        resp = self.client.execute_query('SHOW CREATE EDGE INDEX single_edge_index')
+        self.check_resp_succeeded(resp)
+        expect = [['single_edge_index', 'CREATE EDGE INDEX `single_edge_index` ON `edge_1` (\n `col2`\n)']]
```

Comment:
drop the `single_edge_index `, then use the expect result to create `single_edge_index`